website/spec/v0.1/did-plc.md

```diff
@@ -222,13 +222,62 @@ To fully validate a DID document against the operation log:
     - if no key matches, there has been a trust violation; the PLC server should never have accepted the operation
 - verify the correctness of "nullified" operations and the current active operation log using the rules around rotation keys and recovery windows
 
-The complete log of operations for all DIDs on the PLC server can be enumerated efficiently:
 
-- HTTP endpoint: `https://plc.directory/export`
-- output format: [JSON lines](https://jsonlines.org/)
-- `count` query parameter, as an integer, maximum 1000 lines per request
-- `after` query parameter, based on `createdAt` timestamp, for pagination
+### Bulk Export
 
+The complete log of operations for all DIDs on the PLC server can be enumerated efficiently.
+
+The HTTP endpoint `https://plc.directory/export` returns data in [JSON lines](https://jsonlines.org/) format. It supports two query parameters:
+
+- `count`: An integer, maximum 1000 lines per request.
+- `after`: The cursor for pagination, either an integer or a timestamp string (see below).
+
+When `after` is an integer, each JSON line has the following fields:
+
+- `type` (string): A string with fixed value `indexed_op`.
```

Comment:
This is a nit and it didn't occur to me while looking at code, but how do you feel about something like `accepted_op` over `indexed_op`?

Reply:
(also then I could refer to it as "the sequenced_op format" instead of non-legacy)

Reply:
i think i like "sequenced" better than "indexed". but we're not really notifying consumers that an op was "sequenced" we're notifying them that it was validated & accepted into the directory.

I also wouldn't mind just `op` or `operation`. although I suppose that ends up looking pretty redundant with the `plc_operation` type on the actual operation itself 🤔